src/graphql-server/resolvers.js

```diff
@@ -11,7 +11,7 @@ export const resolvers = {
   Mutation: {
     registerUser: (_, userData) => dataMappedCall('person', userData, HyloDnaInterface.currentUser.create),
 
-    registerHostingUser: (host_doc) => HhaDnaInterface.currentUser.create(host_doc),
+    registerHostingUser: (_, hostDoc) => HhaDnaInterface.currentUser.create(hostDoc),
```

Comment:
notice the change here. This will have been causing you problems. The first param to the resolver function is a `parent` object, which for these top level queries we typically (never?) need.

As well as writing resolvers for top level queries of Mutation and Query, you can also write resolvers for individual types. That's when the `parent` param becomes useful. But we'll cross that bridge when we come to it